pkg/cmd/cli/cmd/logs.go

```diff
@@ -88,6 +88,7 @@ func NewCmdLogs(name, parent string, f *clientcmd.Factory, out io.Writer) *cobra
 // resource a user requested to view its logs and creates the appropriate logOptions
 // object for it.
 func (o *OpenShiftLogsOptions) Complete(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []string) error {
+	containerName := kcmdutil.GetFlagString(cmd, "container")
 	if err := o.KubeLogOptions.Complete(f.Factory, out, cmd, args); err != nil {
```

Comment:
@liggitt actually I think it was the other func that brreak it, you're right it is part of that.